django-backend/fecfiler/scha_transactions/test_models.py

```diff
@@ -58,7 +58,10 @@ def test_generate_uid(self):
         f3x.save()
         trans.save()
         saved_trans = SchATransaction.objects.get(id=trans.id)
-        self.assertEquals(trans.id, saved_trans.id)
+        self.assertEquals(trans.transaction_id, saved_trans.transaction_id)
```

Comment:
This is not quite testing what we want, because you explicitly call `generate_unique_transaction_id` earlier.  we want to test that it's called on `save`

Reply:
I think that this is no longer an issue?

Reply:
I was mainly trying to recommend what you implemented on line 72 (so that's all good).